features/step_definitions/web_steps.rb

```diff
@@ -1,3 +1,3 @@
 When(/^I click the (.*) "(.*)"$/) do |elem, text|
-  find(elem).find('*', text: text).click
-end
\ No newline at end of file
+  all(elem, text: text, count: 1)[0].click
```

Comment:
using find() instead of all() implicitly validates that only one matching element exists, which should always be the case unless there is a problem.